app/controllers/comments_controller.rb

```diff
@@ -38,6 +43,14 @@ def create
     end
   end
 
+  # def create
+  #  @post = Post.find(params[:post_id])
+  #  @comment = @post.comments.build(params[:comment])
+  #  @comment.user = current_user
+  #  @comment.save
+  #  redirect_to post_path(@post)
+  # end
+
```

Comment:
[Optional]
- [ ] You should never commit commented-out code. That's what version control is for. If you need that code back, just use git log to see commit history.

Reply:
ok